dist/configs/config/server.cfg

```diff
@@ -66,6 +66,11 @@ set g_teamForceBalance 2
 //set g_doWarmup 1
 //set g_warmup 20
 
+// allow to fill teams with bots, can also be used in map rotation for
+// per-map settings, *as long as the map is choosen by normal rotation*,
```

Comment:
What's the point of suggesting something which doesn't work? Map configs exist and are a working solution for this.

Reply:
The point is to warn people that host many maps of the limitations.

> Map configs exist and are a working solution for this.

I'm not sure creating 114 individual files is a good solution though. This number is *not* ridiculous, mind you:

> find www/unvanquished -name "map-*.dpk"  |  cut -f1 -d_ | sort | uniq | wc -l
114

This is what I have. And I expect the number to keep growing.
But if you insist I can remove the comment, if that's the only thing preventing this feature from being merged.

Reply:
Yes please remove the mention of map rotations

Reply:
I removed it